### PR TITLE
[DOCS-7358] Add HR Employee File Management solution to Alfresco Content Accelerator 3.5 docs

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: content-accelerator/latest/install/install-guide.md [hunk 1]
```diff
@@ -308,22 +312,47 @@ In this section we ensure that all components of the Content Accelerator are ins
...
-3. (Pnp ONLY) This step is only required if installing the Policy and Procedure Content Accelerator solution:
+3. (Pnp and HR ONLY) This step is only required if using the Policy and Procedure Content Accelerator or HR Content Accelerator solution:
 
-   Navigate to the `ALFRESCO_HOME/amps` directory and copy the following amps there:
-   * `tsgrp-alfresco-chain-versioning.amp`
-   * `pnp-platform-3.5.amp`
+   Copy the `WizardAdmin.war` file into the `ALFRESCO_HOME/tomcat/webapps` directory.
 
-   These amps can be found in the alfresco-content-accelerator-policy-and-procedure-accelerator distribution zip under `Alfresco Artifacts` folder.
+   You'll find this WAR file in the `Web Applications` folder of the `alfresco-content-accelerator-policy-and-procedure-accelerator` zip or `alfresco-content-accelerator-sehr-accelerator` zip.
```
Comment (anxumalo) [inline content-accelerator/latest/install/install-guide.md:319]: Need to revert this section & move it to step 3 in `install-guide/#install-webapps` section .. _to both subsections_? 🤔

Comment (anxumalo) [reply content-accelerator/latest/install/install-guide.md]: Step 3 moved by commit [f081bee](https://github.com/Alfresco/docs-alfresco/pull/1141/commits/f081bee19ecff9b89e69d7f1142c7e199a221588)

Comment (anxumalo) [reply content-accelerator/latest/install/install-guide.md]: Both Tomcat subsection updated - see latest commit [651cb0b](https://github.com/Alfresco/docs-alfresco/pull/1141/commits/651cb0b620807d13b39bf8090b9420b96dd5ce35)

File: content-accelerator/latest/install/install-guide.md [hunk 2]
```diff
@@ -307,10 +311,12 @@ In this section we ensure that all components of the Content Accelerator are ins
    * If using Alfresco Content Services 7.1.x, use the `tsgrp-opencontent-3.5-for-acs7.1.amp`.
    * If using Alfresco Content Services 7.2.x, use the `tsgrp-opencontent-3.5-for-acs7.2.amp`.
    * If using Alfresco Content Services 7.3.x, use the `tsgrp-opencontent-3.5-for-acs7.3.amp`.
+   * If using Alfresco Content Services 7.4.x, use the `tsgrp-opencontent-3.5-for-acs7.4.amp`.
```
Comment (anxumalo) [inline content-accelerator/latest/install/install-guide.md:314]: Verify artefact versions

File: _data/toc/content-accelerator.yaml [hunk 3]
```diff
@@ -59,3 +59,5 @@
         path: '/content-accelerator/latest/configure/activewizard/'
       - title: 'Supported Languages'
         path: '/content-accelerator/latest/configure/supported-languages/'
+      - title: 'HR Tier-2 Solution'
+        path: 'content-accelerator/latest/configure/hr-management/'
```
Comment (anxumalo) [inline _data/toc/content-accelerator.yaml:63]: Fix URL

File: content-accelerator/latest/install/install-guide.md (hunk 2 above)
Comment (anxumalo) [reply content-accelerator/latest/install/install-guide.md]: Switched to generic version: `-3.5.x-`

**Review by katiebrill — DISMISSED** [review]

**Review by mitvachhani — APPROVED** [review]

File: content-accelerator/latest/configure/hr-management.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+8. Navigate to `Holds` under `File Plan`, create a `New Hold` (as shown) and save it:
+
+    ![Create a New Hold]({% link content-accelerator/images/hrefm-create-new-hold.png %})
+
+9. Set the `Read and File` permission to `site_hr_SiteManager` and `site_hr_SiteCollaborator` Groups on Above created `HREFM` Hold.
+
+10. Provide the `HR Manager` group the correct permissions to create Legal Holds:
+
+    1. Navigate to `RM Admin Tools` from the top-right of the RM site.
+    2. Click **Define Roles** in the **Tools** section, and then click on `New Role` button to the left of page.
```
Comment (oliveralfresco2018) [inline content-accelerator/latest/configure/hr-management.md:57]:
```suggestion
    2. Click **Define Roles** in the **Tools** section, and then click on the `New Role` button to the left of page.
```

**Review by oliveralfresco2018 — CHANGES_REQUESTED** [review]
one minor change

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

**Pull request merged**